### DISCUSSION TITLE
"Invalid request domain" under blog post.

### DISCUSSION THREAD

**Discussion by yosoyfreeman** [sujaykundu777/devlopr-jekyll#146] [Q&A]
Hi, i never worked on a web before and I'm loving developr. I am getting this warning only under blog posts.

![imagen](https://user-images.githubusercontent.com/64601424/160918798-b925dfb2-bee8-44e3-8f33-384e5fb5bf2c.png)

But i don't really understand what exactly should i do or how.  Thank you for the help!

**Comment by automationxpert** [comment]
i too experienced this and then figured it out. May be you might have figured it as well, but mentioning here for the folks, who end up like me. 

This is related with hyvor talk used in the template. Just comment or remove the hyvor_talk_website_id line in the _config.yml and that should fix this issue.
